mempool/mempool_test.go

```diff
@@ -339,6 +371,184 @@ func (p *poolHarness) CreateTxChain(firstOutput spendableOutput, numTxns uint32)
 	return txChain, nil
 }
 
+// createTx creates a zero-fee regular transaction from the provided spendable
+// output.
+func (p *poolHarness) createTx(out spendableOutput) (*dcrutil.Tx, error) {
```

Comment:
I'd rather not have this at all, but if you're going to add it, please make it consistent with the rest in making it exported from the harness as it's meant to be used by consumers of it.